README.adoc

```diff
@@ -1069,6 +1069,25 @@ User.all.size
 User.all.length
 ----
 
+=== Check if the collection is empty [[check-empty-collection]]
+
+When checking if the unloaded Active Record collection is empty, prefer `any?`/`empty?` over `present?`/`blank?`. The former executes a simple `EXISTS`-like query while the latter loads the whole collection to determine its size.
```

Comment:
[This](https://github.com/rubocop/rails-style-guide/issues/232#issuecomment-425848425) is even more descriptive.
`user.articles.any?` is used just as frequently as `users.where(...).any?`, and in the former case, `any?` behavior is not equivalent to `exists?`, and is not behaving similarly to `empty?`.